package.json

```diff
@@ -7,14 +7,15 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^14.3.0",
+    "classnames": "^2.3.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "^5.0.1"
   },
   "scripts": {
-    "start": "PORT=3007 react-scripts start",
-    "build": "PORT=3007 react-scripts build",
-    "test": "PORT=3007 react-scripts test",
+    "start": "set PORT=3007 && react-scripts start",
```

Comment:
finally i will not do much about this, since, the "set..." synthax is not valid for unix, it's skip it and defaul run the project on port 3000... 🤷🏾‍♂️

Reply:
You can maybe add [Cross-Env](https://www.npmjs.com/package/cross-env) package. I think this can help in this situation.

Reply:
I added and tested it and it works, thank